### PR TITLE
take in account embedded elements for Cfi and first visible

### PR DESCRIPTION
**Pull request by bjdmeest**

### CODE REVIEW

File: src/renderer.js [hunk 1]
```diff
@@ -484,6 +484,16 @@ EPUBJS.Renderer.prototype.firstElementisTextNode = function(node) {
 	return false;
 };
 
+EPUBJS.Renderer.prototype.isGoodNode = function(node) {
+	"use strict";
```
Comment (fchasen) [inline src/renderer.js]: What does "use strict" do here?

Comment (bjdmeest) [inline src/renderer.js]: That's just a directive to make sure you don't, e.g., reference undeclared
global variables etc. in that function (
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Strict_mode).
It's not necessary :)

2015-01-07 20:50 GMT+01:00 Fred Chasen <notifications@github.com>:

Comment (fchasen) [inline src/renderer.js]: I think we should avoid putting it in individual functions as it is already added here (https://github.com/futurepress/epub.js/blob/master/src/base.js#L1) and the files are concatenated when built.

Comment (bjdmeest) [inline src/renderer.js]: Oh yeah, sorry, overlooked the global one :). I'll do a new pull request
without the use strict :)

2015-01-08 21:17 GMT+01:00 Fred Chasen <notifications@github.com>:

**Comment by fchasen** [comment]
This looks great, will build a new release with these today.

**Pull request merged**